lib/src/mock_navigator.dart

```diff
@@ -1,6 +1,27 @@
 import 'package:flutter/material.dart';
 import 'package:mocktail/mocktail.dart';
 
+class _MockMaterialPageRoute extends MaterialPageRoute<void> {
+  _MockMaterialPageRoute({required super.builder});
+
+  void hackOverlays() {
```

Comment:
Great! 

I spent a good few minutes investigating what could have caused my test to fail after updating to Flutter 3.13.0.

I noticed that without using mockingjay, everything worked as expected (just like in version 3.10.0).

Reply:
I create an issue about this, also: https://github.com/VeryGoodOpenSource/mockingjay/issues/52

Reply:
Thanks for the issue @felipecastrosales !!!